src/containers/SettingList/Organisation/Organisation.test.tsx

```diff
@@ -48,14 +67,49 @@ test('it renders component in edit mode', async () => {
   // loading is show initially
   expect(getByText('Loading...')).toBeInTheDocument();
 
+  //correct values are rendered in the form
   await waitFor(() => {
-    const phoneNumber = getByTestId('phoneNumber');
+    const inputElements = screen.getAllByRole('textbox');
+    const numberInputElements = screen.getAllByRole('spinbutton');
+
+    const orgName = inputElements[0] as unknown as HTMLInputElement;
```

Comment:
```suggestion
    const orgName = inputElements[0] as HTMLInputElement;
```
Any reason we are converting it to unknown first?

Reply:
because it is an HTMLElement so it doesn't have value prop

Reply:
But we can directly do `as HTMLInputElement`. Why do we need to add `as unknown`